package.json

```diff
@@ -37,8 +37,13 @@
     "zombie": "^6.1.4"
   },
   "dependencies": {
+    "@backstage/core": "^0.1.1-alpha.4",
     "@types/classnames": "^2.2.9",
-    "moment": "^2.24.0"
+    "@types/react-table": "^7.0.16",
+    "moment": "^2.24.0",
+    "react-table": "^7.0.4",
+    "react-virtualized-auto-sizer": "^1.0.2",
+    "react-window": "^1.8.5"
```

Comment:
none of these here please, should all go in the package using them

Reply:
Woops, sorry. Though they would be added to the correct package automatically when doing yarn add -W <package>. Guess not.